README.md

```diff
@@ -96,10 +96,10 @@ after running `npm run test`.
 To generate JavaScript API reference documentation run:
 
 ```
-npm run doc
```

Comment:
See note below. I think we may want to keep `npm run doc`.

Reply:
Add back npm run doc

Reply:
Add back npm run doc